gallery/package.json

```diff
@@ -25,7 +25,8 @@
     "@types/classnames": "=2.2.7",
     "@types/react": "16.9.14",
     "@types/react-dom": "16.9.1",
-    "@types/react-router-dom": "=4.3.2",
+    "@types/react-router": "^5.1.0",
```

Comment:
Should we be using =5.1.0 instead? With this syntax each build on CI could have a different version of react-router whether we expect it to or not. That also means the CI build could pick up a new version of the dependency with bad code embedded before we've had a change to review things.

Reply:
NVM, this is only in the gallery so I'm not concerned.

Reply:
> Should we be using =5.1.0 instead? 

No, in fact I'm currently working on opening up all of the dependency versions.

> With this syntax each build on CI could have a different version of react-router whether we expect it to or not. That also means the CI build could pick up a new version of the dependency with bad code embedded before we've had a change to review things.

Not true.  The lockfile prevents that.

Reply:
Good point, CI should only be using the lock file so as long as we review the lock file before merging we're fine.

Reply:
> Good point, CI should only be using the lock file so as long as we review the lock file before merging we're fine.

Well, on that topic, the lockfile often changes in mysterious ways.  It's more a question of _did_ it change in a given PR (and it shouldn't unless deps in package.json are modified) rather than _how_ it changed.